src/i18n/content/jp/docs/agents/net-agent/troubleshooting/no-data-appears-net.mdx

```diff
@@ -10,6 +10,8 @@ metaDescription: Troubleshooting steps if data does not appear in your app after
 
 [コンテンツ](/docs) / [APMエージェント](/docs/agents) / [.NETエージェント](/docs/agents/net-agent) / [トラブルシューティング](/docs/agents/net-agent/troubleshooting)
 
+# データが表示されない（.NET） [#page-title]
```

Comment:
i saw a couple of these. don't think this heading is supposed to be there. if you look at the English file, it doesn't have this. and troubleshooting docs really should only have Problem, Solution, Cause as possible headings.